azext_iot/tests/digitaltwins/__init__.py

```diff
@@ -422,6 +422,20 @@ def ensure_servicebus_resource(self):
                 )
             )
 
+    def ensure_user_identity(self):
+        """Create a user identity to use with the test. Will populate self.user_identity
+
+        Returns the user identity object."""
+        if not hasattr(self, "user_identity_name"):
+            self.user_identity_name = generate_resource_id()
+            return self.embedded_cli.invoke(
+                f"identity create -n {self.user_identity_name} -g {self.rg}"
+            ).as_json()
+        else:
+            return self.embedded_cli.invoke(
+                f"identity show -n {self.user_identity_name} -g {self.rg}"
```

Comment:
Does the show not need to be returned `.as_json()`?

Reply:
I blame my past self

Reply:
lmao